input/kafkamdm/kafkamdm.go

```diff
@@ -251,6 +254,10 @@ func (k *KafkaMdm) consumePartition(topic string, partition int32, currentOffset
 			if err := offsetMgr.Commit(topic, partition, currentOffset); err != nil {
 				log.Error(3, "kafka-mdm failed to commit offset for %s:%d, %s", topic, partition, err)
 			}
+			if currentOffset < 0 {
+				// we have not yet consumed any messages.
+				continue
```

Comment:
How about the scenario where new metrics get added to kafka but we didn't consume them yet.
would it make sense to call `k.client.GetOffset(topic, partition, sarama.OffsetNewest)` and update our metrics?

I suspect it would be unlikely that we're not consuming messages but that such a call would work fine (and return quickly)